content/en/docs/ambient/install/multicluster/multi-primary_multi-network/index.md

```diff
@@ -0,0 +1,412 @@
+
+{{< text bash >}}
+$ helm install istiod istio/istiod -n istio-system --kube-context "${CTX_CLUSTER1}" --set global.meshID=mesh1 --set global.multiCluster.clusterName=cluster1 --set global.network=network1 --set profile=ambient --set pilot.env.AMBIENT_ENABLE_MULTI_NETWORK="true"
+{{< /text >}}
+
+{{< /tab >}}
+
+{{< /tabset >}}
+
+## Install an ambient east-west gateway in `cluster1`
+
+Install a gateway in `cluster1` that is dedicated to ambient
+[east-west](https://en.wikipedia.org/wiki/East-west_traffic) traffic. By
+default, this gateway will be public on the Internet. Production systems may
+require additional access restrictions (e.g. via firewall rules) to prevent
+external attacks. Check with your cloud vendor to see what options are
+available.
+
+### Deploy Gateway API CRDs
+
+{{< text syntax=bash snip_id=install_crds >}}
+$ kubectl get crd gateways.gateway.networking.k8s.io --context="${CTX_REMOTE_CLUSTER}" &> /dev/null || \
+  { kubectl kustomize "github.com/kubernetes-sigs/gateway-api/config/crd?ref={{< k8s_gateway_api_version >}}" | kubectl apply -f - --context="${CTX_REMOTE_CLUSTER}"; }
+{{< /text >}}
+
+### Install the east-west gateway
+
+{{< tabset category-name="east-west-gateway-install-type-cluster-1" >}}
+
+{{< tab name="IstioOperator" category-value="iop" >}}
+
+{{< text bash >}}
+$ @samples/multicluster/gen-eastwest-gateway.sh@ \
+    --network network1 \
+    --ambient | \
+    istioctl --context="${CTX_CLUSTER1}" install -y -f -
+{{< /text >}}
+
+{{< warning >}}
+If the control-plane was installed with a revision, add the `--revision rev` flag to the `gen-eastwest-gateway.sh` command.
+{{< /warning >}}
+
+{{< /tab >}}
+{{< tab name="Kubectl apply" category-value="helm" >}}
+
+Install the east-west gateway in `cluster1` using the following Gateway definition:
+
+{{< text bash >}}
+$ cat <<EOF > cluster1-ewgateway.yaml 
+kind: Gateway
+apiVersion: gateway.networking.k8s.io/v1
+metadata:
+  name: istio-eastwestgateway
+  namespace: istio-system
+  labels:
+    topology.istio.io/network: "network1"
+spec:
+  gatewayClassName: istio-east-west
+  listeners:
+  - name: mesh
+    port: 15008
+    protocol: HBONE
+    tls:
+      mode: Terminate # represents double-HBONE
+      options:
+        gateway.istio.io/tls-terminate-mode: ISTIO_MUTUAL 
+EOF
+{{< /text >}}
+
+{{< warning >}}
+TODO: Handle revisions
+{{< /warning >}}
+
+Apply the configuration to `cluster1`:
+
+{{< text bash >}}
+$ istioctl install --context="${CTX_CLUSTER1}" -f cluster1-ewgateway.yaml
+{{< /text >}}
+
+{{< /tab >}}
+
+{{< /tabset >}}
+
+Wait for the east-west gateway to be assigned an external IP address:
+
+{{< text bash >}}
+$ kubectl --context="${CTX_CLUSTER1}" get svc istio-eastwestgateway -n istio-system
+NAME                    TYPE           CLUSTER-IP    EXTERNAL-IP    PORT(S)   AGE
+istio-eastwestgateway   LoadBalancer   10.80.6.124   34.75.71.237   ...       51s
+{{< /text >}}
+
+## Expose services in `cluster1`
```

Comment:
Thoughts on moving this to the end once everything is configured?

Reply:
Agreed - will move

Reply:
Verify has this step, so I just removed it entirely from this page.